src/settings/SettingsModal.tsx

```diff
@@ -95,6 +102,30 @@ const SettingsModal = ({
                 onChange={onChangeTooltipsToggle}
               />
             </FormControl>
+            <RenderIf shouldRender={getBoolFromEnv('VITE_DEBUG_MODE')}>
+              <Text as="b">Scene Navigation</Text>
+              <Grid display="flow">
+                {Object.keys(SceneName)
+                  .filter(
+                    (key) =>
+                      isNaN(Number(SceneName[key as keyof typeof SceneName])) &&
```

Comment:
Can we create a `getEnumStringKeys` method in `src/common/utils/get-enum-string-keys`? I think this will make the logic easier to understand later when we forget that Object.keys on the enum returns both the numbers and strings.

Reply:
Also, I think you should be able  to just do `isNaN(Number(key))`, could be wrong though.

Reply:
Also you may be able to do something fancy with generics where the returned keys are `keyof typeof {Input enum}` so you don't have to keep casting it later